romancal/dq_init/dq_initialization.py

```diff
@@ -0,0 +1,147 @@
+import logging
+
+import numpy as np
+
+from .. import datamodels
+#from ..lib import reffile_utils
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+# Guide star mode exposure types
+guider_list = ['WFI_WIM_ACQ','WFI_WIM_TRACK','WFI_WSM_ACQ1','WFI_WSM_ACQ2','WFI_WSM_TRACK']
+
```

Comment:
These modes look fine to me and match the guide window table 3.5.2.1 Table of Guide Window Sizes and Use.

Reply:
Is the intermediate function correct_model() needed ? It consists of a single statement calling another function having the same arguments. Perhaps I'm missing something here.

Reply:
The code in `do_dqinit` assumes these values are enumerated in `meta.exposure.type` .